src/Foundatio/Messaging/IMessageBus.cs

```diff
@@ -2,4 +2,9 @@
 
 namespace Foundatio.Messaging {
     public interface IMessageBus : IMessagePublisher, IMessageSubscriber, IDisposable {}
+    
+    public class MessageOptions {
+        public string UniqueId { get; set; }
```

Comment:
Let's add a CorrelationId and Properties on here like QueueEntryOptions has.

Reply:
Done, although I'm not sure if I need to use them in any of the implementations.

Reply:
That's fine. We should be able to support them in a few of the systems. We can add that implementation.